docs/blitz/design/actions/yang.rst

```diff
@@ -100,17 +100,21 @@ subscription, or, you may expect the test to fail (referred to as a negative tes
 .. code-block:: YAML
 
     format:
-      request_mode:    # [STREAM, ONCE, POLL] gNMI subscription mode
-      sub_mode:        # [ON_CHANGE, SAMPLE] gNMI subscription sub_mode
-      encoding:        # [JSON, JSON_IETF, PROTO, ASCII] gNMI val encoding
-      prefix:          # [true | false] gNMI message requires PATH prefix
-      origin:          # [openconfig | rfc7951 | module | <device defined> ] gNMI origin
-      base64:          # [true | false] gNMI set "val" requires Base64 encoding
-      sample_interval: # number of seconds between sampling
-      stream_max:      # seconds to stop stream (default: 0, no max)
-      auto-validate:   # [true | false] automatically validate config messages
-      negative-test:   # [true | false] expecting device to return an error
-      pause:           # pause N seconds between each test (default: 0, no pause)
+      request_mode:     # [STREAM, ONCE, POLL] gNMI subscription mode
+      sub_mode:         # [ON_CHANGE, SAMPLE] gNMI subscription sub_mode
+      encoding:         # [JSON, JSON_IETF, PROTO, ASCII] gNMI val encoding
+      prefix:           # [true | false] gNMI message requires PATH prefix
+      origin:           # [openconfig | rfc7951 | module | <device defined> ] gNMI origin
+      base64:           # [true | false] gNMI set "val" requires Base64 encoding
+      sample_interval:  # number of seconds between sampling
+      stream_max:       # seconds to stop stream (default: 120, no max)
+      auto-validate:    # [true | false] automatically validate config messages
+      negative-test:    # [true | false] expecting device to return an error
+      pause:            # pause N seconds between each test (default: 0, no pause)
+      transaction_time: # number of seconds that determines the maximum time that can pass between sending a request and receiving a response
+      polls_number:     # only for POLL subscribtions, determines how many polls should be sended (default: stream_max // sample_interval)
+      updates_only:     # [true | false] only for Subscribe requests, determines if only updates should be received (default: false)
+      continue_on_failure: # [true | false] continue test if a step fails (default: false)
```

Comment:
I see you added a ``continue_on_failure``.  This is not good.  pyats tests continue to execute now.  Having this option is confusing.  Find another way to fix whatever issue you are trying to solve without adding another flag from user.